DevArt.Users.Application/Dto/HttpRequestMessageOptionDto.cs

```diff
@@ -0,0 +1,12 @@
+namespace DevArt.Users.Application.Dto;
+
+public class HttpRequestMessageOptionDto
+{
+    public HttpMethod Method { get; set; }
+    
+    public string Route { get; set; }
+    
+    public Object Body { get; set; }
```

Comment:
Can we make this to be a generic type?
```suggestion
    public T Body { get; set; }
```

Reply:
Of course, I will update it and test it again. Please wait me a little bit

Reply:
Because `JsonContent.Create` can accept a generic value. As a result, we change the type of body to generic type.
![image](https://github.com/user-attachments/assets/7a6d379d-1ab9-4d3c-9128-eaa8c5f6aedf)